changelog/1591.trivial.rst

```diff
@@ -1 +0,0 @@
-Fixed warnings in plasmapy/particles/particle_class.py (deprecated escape sequence).
```

Comment:
Why did you remove the changelog entry?

Reply:
This PR was changing a `"""` to `r"""` for a docstring, so we can get away without one.